admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/validator/SpecialCharacterValidator.java

```diff
@@ -0,0 +1,30 @@
+package io.mosip.kernel.masterdata.validator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+import org.springframework.beans.factory.annotation.Value;
+
+import lombok.Data;
+
+@Data
+public class SpecialCharacterValidator implements ConstraintValidator<CharacterValidator, String> {
+
+	@Value("${allowed_characters:a-z0-9}")
```

Comment:
follow the convention like mosip.kernel.masterdata.auth-user-details=/userdetails and give proper name for this property
mosip.kernel.masterdata.code.validate.regex